packages/ag-charts-website/src/content/docs/charts-nightingale-series/index.mdoc

```diff
@@ -0,0 +1,145 @@
+---
+title: "Nightingale Series"
+enterprise: true
+---
+
+**Nightingale** series (sometimes called a *Rose chart*) represent a dataset in a form of a polar area diagram.
+
+## Grouped Nightingale
+
+To plot a basic Nightingale we need an array of data, where every item will have a single category value (for the angle axis) and a single numeric value (for the radius axis).
+
+A minimal nightingale series configuration will look like:
```

Comment:
Capitalise Nighingale maybe for consistency with the sentence above or have it in quotes to represent the type name `'nightingale'`?